pkgs/by-name/ze/zed-editor/package.nix

```diff
@@ -137,50 +136,11 @@ rustPlatform.buildRustPackage rec {
 
   postInstall = ''
     mv $out/bin/Zed $out/bin/zed
-    install -D ${src}/crates/zed/resources/app-icon@2x.png $out/share/icons/hicolor/1024x1024@2x/apps/Zed.png
-    install -D ${src}/crates/zed/resources/app-icon.png $out/share/icons/hicolor/512x512/apps/Zed.png
+    install -D ${src}/crates/zed/resources/app-icon@2x.png $out/share/icons/hicolor/1024x1024@2x/apps/zed.png
+    install -D ${src}/crates/zed/resources/app-icon.png $out/share/icons/hicolor/512x512/apps/zed.png
+    install -Dm644 ${src}/crates/zed/resources/zed.desktop -t $out/share/applications/dev.zed.Zed.desktop
```

Comment:
Something went wrong with the desktop file install location...
```
$ ls --tree result
result ⇒ /nix/store/yymqkjsc84d3bk4bfn3zrz99c687y989-zed-0.134.4
├── bin
│   └── zed
└── share
    ├── applications
    │   └── dev.zed.Zed.desktop
    │       └── zed.desktop
```

Reply:
Indeed, any idea why it behaves like this ?

Reply:
Seems good now.
The `-t` option of `install` means "target directory".